app/templates/organization/settings.hbs

```diff
@@ -0,0 +1,18 @@
+<section class='organization-settings'>
+  {{#if (and features.proVersion organization.permissions.admin)}}
```

Comment:
I would suggest to move this condition to controller giving it a name something like `showOrganizationSettings`.

Reply:
Yeah that sounds good to me. Change made :)